<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SemanticPlannerDlg</class>
 <widget class="QDialog" name="SemanticPlannerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="codebookFilePathInlineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>C:\output_data\graspit_test_scans\guidebook504s125.txt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadScanbookButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QLabel" name="codebookStatusLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Loaded: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="semanticGraspPathInlineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>c:\output_data\graspit_test_scans\sg_template.txt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadSemanticGraspsButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load SG's</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>351</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Steps</string>
   </property>
   <widget class="QPushButton" name="nextNeighborButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setHandButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Hand</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adjustHandButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Adjust</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previousNeighborButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Neighbor</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>N_id: N_LL</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="numberOfVotingNeighbors">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="voteButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Vote</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="phiLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="thetaLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bestButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Best</string>
    </property>
   </widget>
   <widget class="QPushButton" name="encodeButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Encode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refineButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Refine</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scanButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getNNButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get N.N.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>||</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previousButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="solutionLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0/0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rankButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Rank</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showArrowCheckBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>arrow</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="SGStatusLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Loaded: </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>201</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Utils</string>
   </property>
   <widget class="QPushButton" name="tactileButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Print Tactile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="storeButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>60</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Store SG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showExamplePreGraspButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SGFileNameInlineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>sg.txt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rotLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>90</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bestIdxLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>45</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>roll</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>32</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>index</string>
    </property>
   </widget>
   <widget class="QPushButton" name="similarityMeasureButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cal Similarity</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>331</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Example SG ID:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="planButton">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>329</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Plan</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bestIdxLineEdit_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Execution</string>
   </property>
   <widget class="QPushButton" name="executeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>603</x>
     <y>30</y>
     <width>20</width>
     <height>401</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>331</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>Shape correspondence</string>
   </property>
   <widget class="QLineEdit" name="correspondenceFilePathInlineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C:\project\graspit_hao\semantics\correspondence_wine_glass_401.txt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadCorrespondenceButton">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Cor.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadSGCorrButton">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load SG's</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SGCorrFilePathInlineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C:\project\graspit_hao\worlds\shape_corr\wine_glass.xml</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showCorrespondenceButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Corr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="computeTransformButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute T</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SemanticPlannerDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SemanticPlannerDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
